### ISSUE TITLE
Wage match scrubber enhancement: update IEVP & add case note 

### ISSUE THREAD

**Issue by IlseFerris** [MN-Script-Team/DHS-MAXIS-Scripts#2634]
feedback from staff using the wage match scrubber. Could the script update IEVP and add case note actions if worker can resolve issue or need follow up steps.

**Comment by CDPotter** [comment]
My gut says this may not be the best option, if only to avoid another CSES scrubber situation.  I'm open to hearing other thoughts on the matter.

**Comment by RoberT-1000** [comment]
I agree with @CDPotter ... we had the same input in Anoka but it was rejected. We wanted staff updating IEVP. At that point, if the script is not updating anything, it should not be case noting anything.

Question (because I haven't done this in ages): if IEVP is updated and a match is cleared, is it available later or has it been cleared? (I ask because if that's the case, I am concerned that an accidental script usage could wipe a wage match from future processing.)

**Comment by Farleight** [comment]
The match itself disappears, but the info is still available IF you know to look for it... which you wouldn't.

**Comment by C-Love** [comment]
I think this is great. I think it should be an entirely worker agency - basically the script does NO guessing on HOW it should be filled out. Basically just all the IEVP options. then we can used that dialog for some additional information that is not available on IEVP.
